packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -117,11 +117,14 @@ class _InspectorReferenceData {
 ///
 /// All methods returning String values return JSON.
 class WidgetInspectorService {
-  WidgetInspectorService._();
+  @visibleForTesting
```

Comment:
this will need documentation.

BTW, @visibleForTesting doesn't seem to really do anything. So really this just makes it public.

Reply:
I've followed the suggestion to make this class only work as a mixin which seems reasonable given @visibleForTesting isn't doing anything really useful yet.